### PR TITLE
Added the DateManager class

### PR DESCRIPTION
**Pull request by tzavidas**

### CODE REVIEW

**Review by RemusNeatu — APPROVED** [review]

File: src/main/webapp/utilities/SessionManager.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+    /**
+     * Gets JSON-parsed value from LocalStorage
+     * @returns {Any} the value on that field
+     */
+    getValueInLocalStorageParsed() {
+        return JSON.parse(this.localStorage.getItem(this.localStorageKey));
+    }
+
+    /**
```
Comment (maxwbuckley) [inline src/main/webapp/utilities/SessionManager.js:16]: nit: writeValueToLocalStorage?

File: src/main/webapp/utilities/SessionManager.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+     * Sets value to LocalStorage
```
Comment (maxwbuckley) [inline src/main/webapp/utilities/SessionManager.js]: Add to the docstring that this will overwrite the existing value if called again

**Review by maxwbuckley — APPROVED** [review]

File: src/main/webapp/utilities/SessionManager.js (hunk 1 above)
Comment (tzavidas) [reply src/main/webapp/utilities/SessionManager.js:16]: Changed it!

File: src/main/webapp/utilities/SessionManager.js (hunk 2 above)
Comment (tzavidas) [reply src/main/webapp/utilities/SessionManager.js]: Done! :)

**Pull request merged**